### ISSUE TITLE
Feature: ability to set innerHTML beyond numeric values

### ISSUE THREAD

**Issue by Cerchie** [juliangarnier/anime#870]
Note: Previously reported in [issue #710](https://github.com/juliangarnier/anime/issues/710) 

I don't think this is a bug since `innerHTML` was not designed with non-numeric values in mind, but it's a feature that would be helpful: the ability to set an element's inner HTML with non-numeric values from a keyframe, like so:

```
	var animationGoToDefReq = anime({
        targets:  '.els',
        keyframes: [
            {translateY: 150},
	    {innerHTML: 'some words here'},
            {translateY: -2},
        ],	
        delay: function(el, i) { return i * 400; },
        direction: 'normal',
        loop: true,
        autoplay: true,
        easing: 'easeInSine',
        duration: 4000
});
```

Currently, `anime.js` adds a zero to whatever content I attempt to add to innerHTML. 

Example: If I try to set an `innerHTML` of 'Location' I see a zero tacked to the end. 
<img width="378" alt="Screenshot 2023-11-09 at 1 28 49 PM" src="https://github.com/juliangarnier/anime/assets/54046179/845788e7-fbbb-422c-a6ad-9a9e10f39d24">

To reproduce: 

```
git clone https://github.com/Cerchie/svelte-scrollytale.git  && checkout innerHTML-issue && npm install && npm run dev
```

(scroll down to where you can see the animation with the zero)

**Comment by juliangarnier** [comment]
Yeah the added `0` to a string value is a odd behaviour, but V3 does't expect non numerical value within an animation.
I might change that in V4, and also allow to "animate" values like `'display'` or `'visibility'`.

Currently in V4 you can use a timeline for that, since `timeline.set(target, { innerHTML: 'som text' })` works.

**Comment by juliangarnier** [comment]
Your issue should be fixed in [v4](https://animejs.com/).

Check out the new [documentation](https://animejs.com/documentation/) and the [migration guide from v3](https://github.com/juliangarnier/anime/wiki/Migrating-from-v3-to-v4).

I'm closing this for now, but feel free to re-open if this is still an issue for you in v4 by following the [contributing guidelines](https://github.com/juliangarnier/anime/blob/master/CONTRIBUTING.md).
